src/app/components/cells/Cell.jsx

```diff
@@ -182,7 +183,9 @@ class Cell extends React.Component {
 
   userCanEditValue() {
     const { cell, langtag } = this.props;
-    return canUserChangeCell(cell, langtag);
+    const langtagOrCountry =
+      cell.kind === "currency" ? getCountryOfLangtag(langtag) : langtag;
```

Comment:
```suggestion
      f.propEq(["column", "languageType"], "country")(cell) ? getCountryOfLangtag(langtag) : langtag;
```

Es ist zwar unwahrscheinlich, aber nicht unmöglich, dass wir noch andere länderspezifische Spalten hereinbekommen. Z. B. verschiedene Element auf Websites in verschiedenen Ländern. Deswegen in solchen Fällen lieber den Sprachtyp überprüfen.